core/src/main/boilerplate/com/softwaremill/diffx/TupleInstances.scala.template

```diff
@@ -0,0 +1,21 @@
+package com.softwaremill.diffx
+
+trait TupleInstances {
+
+  [2..#def dTuple1[[#T1#]](implicit [#d1: Diff[T1]#]): Diff[Tuple1[[#T1#]]] = new Diff[Tuple1[[#T1#]]] {
```

Comment:
this could be 
```
  [2..#implicit def dTuple2[[#T1: Diff#]] = Diff.derived[Tuple2[[#T1#]]]#
  ]
```
instead, but it is generated code anyway so it should be as optimized as possible. 
Also by handcrafting diff instances for tuples we gain the possibility of adjusting that behavior in the future.